src/locales/en.json

```diff
@@ -683,5 +683,36 @@
         "caption": "On my phone"
       }
     }
+  },
+
+  "LoanProgress": {
+    "reimbursedAmount": "Reimbursed amount",
+    "borrowedAmount": "Borrowed amount"
+  },
+
+  "LoanDetails": {
+    "keyInfos": {
+      "title": "Key information",
+      "borrowedAmount": "Borrowed amount:",
+      "remainingCapital": "Remaining capital:",
+      "interestRate": "Interest rate:"
+    },
+    "payments": {
+      "title": "Payments",
+      "lastPayment": "Last payment made:",
+      "nextPayment": "Next scheduled payment:"
+    },
+    "characteristics": {
+      "title": "Loan characteristics",
+      "subscriptionDate": "Subscription date:",
+      "maturityDate": "Repayment date:",
+      "nbPaymentsLeft": "Nb payments left:",
+      "nbPaymentsDone": "Nb payments done:"
```

Comment:
It seems No is the correct abbreviation for number: https://english.stackexchange.com/questions/43095/what-is-the-correct-abbreviation-for-the-word-numbers. I think # is also used.